x-pack/legacy/plugins/apm/index.ts

```diff
@@ -124,13 +131,27 @@ export const apm: LegacyPluginInitializer = kibana => {
           },
           read: {
             app: ['apm', 'kibana'],
-            api: ['apm', 'actions-read', 'alerting-read'],
+            api: [
+              'apm',
+              'actions-read',
+              'actions-all',
+              'alerting-read',
+              'alerting-all'
```

Comment:
this means that our read users can create alerts as well, is that intended/necessary?

Reply:
We're recommending solution teams to use an all or nothing approach from the privileges perspective until we have proper support for feature controls (#43994). It opens up weird cases if we allow granular control. A user would just need `all` in APM, Uptime, Metrics or SIEM to undermine a `read` level alerting access. We could remove this code for readonly APM users, would that make sense?

Reply:
Ah, it's fine then AFAIC to use all-or-nothing.